src/main/java/mx/paradigmmasters/bastagui/view/GameView.java

```diff
@@ -30,6 +30,11 @@ public GameView(MainControl mainControl) throws HeadlessException {
         this.setDefaultCloseOperation(EXIT_ON_CLOSE);
         this.setVisible(true);
         this.setLayout(new GridLayout(4,1));
+        this.setSize(400, 700);
```

Comment:
No es necesario poner un tamaño de ventana, al final del constructor se usa `this.pack()` para que Swing calcule automáticamente el tamaño de ventana ideal, por lo que esta línea es inutil.

Reply:
Pero si dejo que se adapte, es muy pequeña la ventana, hay que agrandarla manualmente y lo que quiero es que sea más cómoda.

Reply:
Entonces es deseable eliminar el método al final del constructor, no pueden convivir al mismo tiempo, primero la ventana se construye con un tamaño fijo y luego `.pack()` lo ajusta.